src/webgpu/api/validation/vertex_access.spec.ts

```diff
@@ -1,91 +0,0 @@
-export const description = `
-TODO: make sure this isn't already covered somewhere else, review, organize, and implement.
-> - In encoder.finish():
->     - setVertexBuffer and setIndexBuffer commands (even if no draw):
->         - If not valid at draw time, test overlapping {vertex/vertex,vertex/index}
->           buffers are valid without draw.
->         - Before, after setPipeline (should have no effect)
->         - Implicit offset/size are computed correctly. E.g.:
->             { offset:         0, boundSize:         0, bufferSize: 24 },
->             { offset:         0, boundSize: undefined, bufferSize: 24 },
->             { offset: undefined, boundSize:         0, bufferSize: 24 },
->             { offset: undefined, boundSize: undefined, bufferSize: 24 },
->             { offset:         8, boundSize:        16, bufferSize: 24 },
->             { offset:         8, boundSize: undefined, bufferSize: 24 },
->         - Computed {index, vertex} buffer size is zero.
->             (Omit draw command if it's not necessary to trigger error, otherwise test both with and without draw command to make sure error happens at the right time.)
->             { offset: 24, boundSize: undefined, bufferSize: 24, _ok: false },
->         - Bound range out-of-bounds on the GPUBuffer. E.g.:
->             - x= offset in {0,8}
->             - x= boundSize in {8,16,17}
->             - x= extraSpaceInBuffer in {-1,0}
->     - All (non/indexed, in/direct) draw commands
->         - Same GPUBuffer bound to multiple vertex buffer slots
```

Comment:
On second thought, we should have an extremely basic test of overlap between two vertex buffers, or overlap between a vertex buffer and an index buffer. I don't think it's likely to find any issues, but it should be very simple to add. It doesn't have to be an operation test - that would be more complicated - we should just submit the work to make sure it doesn't hit a validation error or a backend validation layer (e.g. d3d12 debug layer) error.

Reply:
Got it. Where should this test be? maybe `draw_direct.spec.ts`?

Reply:
I think a new file `draw.spec.ts` would be reasonable, since we could test this for all four draw calls. pseudocode:

`.cases('drawType', all four draw types)`

```
const buf = create buffer (vertex | index | indirect) with some contents;

setVertexBuffer(0, buf)
setVertexBuffer(1, buf);
setIndexBuffer(buf);

do draw of appropriate type (with buf as indirect buffer if any)
```

You don't need to add this in this PR though, just add a TODO for it.

Reply:
Done.